.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Bug report
+about: Tell about typedoc's bug
+---
+<!--
+    Thank you for contributing to Typedoc! 
+    
+    First, please use a clear and descriptive title for the issue to identify the bug.
+
+    Please be as specific as possible! Provide codes, sreenshots, repositories, explanations, issues, links ... anything that could describe your point.
+-->
+
+- [ ] I have checked [issues with bug label](https://github.com/TypeStrong/typedoc/labels/bug) and found no duplicates
+
+## Expected Behavior
+<!--
+    According to the code or documentation, how is Typedoc supposed to work?
+-->
+
+## Actual Behavior
+<!--
+    What does Typedoc fail to do?  
+-->
+
+## Steps to reproduce the bug
+<!--
+    Tell us how we could reproduce the bug. 
```

Comment:
Could we add some suggestions in this comment? How `typedoc` is invoked, configuration options, code that does not get documented correctly, minimal project to reproduce the issue.

Reply:
Sure. I will add that.